src/EmailFactory.php

```diff
@@ -29,8 +29,8 @@ public function create(string $mailbox, Horde_Imap_Client_Data_Fetch $hordeEmail
         // Parse the message body
         $parser = new Parser();
         $parser->setText($hordeEmail->getFullMsg());
-        $htmlContent = utf8_encode((string) $parser->getMessageBody('html'));
```

Comment:
> merge it and see what happens :D
<img width="398" alt="3b44c5" src="https://cloud.githubusercontent.com/assets/154256/24602278/6ccfd234-185c-11e7-9c6b-3d2a36916096.png">